src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+    alt="View your core web vitals data"
+    src={cwvWPM}
+/>
+
+Google PageSpeed reports your core web vitals alongside other metrics collected by New Relic. This data gives you insight into how users experience your site. 
+
+* **First contentful paint**. This captures the time it takes for the first asset to load on a page. In the above screenshot, it took the site 3.22s for something to load on the page. 
+* **Largest contentful paint**. Unlike first contentful paint, this captures the time taken for the main content on the page to fully load. For example, if information about an important campaign appears in a page's main asset, then this number tells us how long a user waits before seeing it. In the above screenshot, it took 2.79s. 
+* **First input delay**. When a user interacts with an element on your site, it takes time to process the user's interaction This number captures that time as a delay between the request (user interaction) and response (how the page changes based on the user request).
+* **Cumulative layout shift**. This measures visual stability of different assets on your site. For example, after a user goes to your site's main page, images may shift while loading. This movement is unanticipated.
+
+<img
+    title="Adjust timeslice data"
+    alt="Adjust timeslice data"
+    src={timeWPM}
+/>
+
+New Relic contextualizes this data over time. You can click and drag over relevant parts of this chart, which gives you more granular insight about site behavior. When you're finished, click the time stamp button at the top to adjust back to your preferred time period. 
+
+    </Collapser>
+    <Collapser
+        id="content"
+        title="Compare content size, load times, and sources"
+    >
+<img
+    title="Analyze content size"
+    alt="Analyze content size"
+    src={contentWPM}
+/>
+
+WPM lets you visualize how different types of assets perform relative to each other. In the **Core web vitals** section, you can view four bar graphs that show your content's size, load time, their resources, and how the network affects them. Using two or more of these graphs can help you identify if, say, your CSS assets are causing load issues. 
+
+If page load speed is slow, then you can use the **Content** section to diagnose potential sources of the problem relative to each other. For example, comparing **Content size** to **Content load time** can give insight into whether CSS is taking up too many resources and loading last, which would cause a delay in largest contentful paint. 
+
+    </Collapser>
+    <Collapser
+        id="geo"
+        title="Fix broken links"
+    >
+
+Unlike a website grader, WPM lets you view your critical pages in context of each other. If user engagement is down but your site is available and content is optimized, then something else might be at play. With WPM, your monitors routinely ping your chosen pages and report back. A possible journey to troubleshoot might look like:
```

Comment:
This is more of an experimental section that I wanted to temp check with @seemantk and @stumblefiend. Should I add an example user journey for other sections, just to flesh out how someone might troubleshoot with WPM?

Reply:
This don't sound correct to me "Unlike a website grader, WPM lets you view your critical pages in context of each other." we don't compare multiple pages in context of each other. All pages get their own dashboard pages.

Reply:
We actually differentiate traditional website graders in the following ways:
- WPM is continuous (website graders are discrete, single snapshot)
- WPM can monitor multiple pages (website graders only allow one page)
- WPM provides alerts for free (website graders dont have alerts)
- WPM provides broken links (website graders dont)
- WPM is free upto 500 checks, 10K with CC, and half a cent for overages.
- WPM offers deep integration into observability ecosystem within NR1 platform.